ClassMoodApp/static/tsc/app.ts

```diff
@@ -18,6 +18,18 @@ module ClassMoodApp {
         })
         .when("/liveView", {
             templateUrl: "/templates/liveView.html"
+        })
+        .when("/lectureList/:classId", {
+            templateUrl: "templates/lectureList.html",
+            controller: "LectureListController"
+        })
+        .when("/pollingQuestionList/:class_id/:lecture_id", {
```

Comment:
Should I just delete all of the entries in this file that relate to template rendering then?